ai-service/src/main/resources/application.yml

```
@@ -0,0 +1,49 @@
+spring:
+  application:
+    name: ai-service
+
+  kafka:
+    bootstrap-servers: localhost:9097 # 로컬 카프카 주소 (환경에 맞게 수정)
```

YAMLlint (1.38.0)

[warning] too few spaces before comment: expected 2

(comments)